primitives/src/locktime/relative.rs

```diff
@@ -77,10 +77,19 @@ impl LockTime {
     /// ```rust
     /// # use bitcoin_primitives::relative::LockTime;
     ///
-    /// // `from_consensus` roundtrips with `to_consensus_u32` for small values.
-    /// let n_lock_time: u32 = 7000;
-    /// let lock_time = LockTime::from_consensus(n_lock_time).unwrap();
+    /// // Values of 0xffff and lower will roundtrip with `to_consensus_u32`.
+    /// let n_lock_time: u32 = 0xffff;
+    /// let lock_time = LockTime::from_consensus(n_lock_time)?;
     /// assert_eq!(lock_time.to_consensus_u32(), n_lock_time);
+    ///
+    /// // Values above 0xffff will be truncated and not round-trip with `to_consensus_u32`.
```

Comment:
In be07b07089a0c315c566d46964b8775b1e47063b:

This is untrue. We also preserve bit 22.

Why are these docs trying to be so detailed about this? Nobody should be walking on eggshells trying to make these types roundtrip. If they need roundtrippability then they can't use the relative locktime types as part of that roundtrip.

Reply:
I have changed the example to be more useful and show the usage with both a height and time based lock time.

Reply:
I think the doc for this function and `from_sequence` should be improved, but will leave that as part of issue #3970.